artifactory/commands/utils/result.go

```diff
@@ -31,3 +39,50 @@ func (r *Result) SetFailCount(failCount int) {
 func (r *Result) SetReader(reader *content.ContentReader) {
 	r.reader = reader
 }
+
+func UnmarshalDeployableArtifacts(filePath string) (*Result, error) {
+	modulesMap, err := jsonFileToModulesMap(filePath)
+	if err != nil {
+		return nil, err
+	}
+	// Iterate map for : counting seccesses/failures & save artifact's SourcePath, TargetPath and Sha256.
```

Comment:
// Iterate map for : counting seccesses/failures & save artifact's SourcePath, TargetPath and Sha256.
-->
// Iterate over the modules map , counting seccesses/failures & save artifact's SourcePath, TargetPath and Sha256.